playwright/tests/standalone/viewport/accessibility.spec.ts

```diff
@@ -0,0 +1,57 @@
+import test, { expect } from '@playwright/test';
+import { ProcessEditor } from '../../page-objects/editor/process-editor';
+
+test('focus tool bar shortcut', async ({ page }) => {
+  await ProcessEditor.openProcess(page);
+  const defaultToolsButton = page.locator('#btn_default_tools');
+  await expect(defaultToolsButton).not.toBeFocused();
+  await page.keyboard.press('Control+Digit1');
+  await expect(defaultToolsButton).toBeFocused();
+});
+
+test('focus diagram shortcut', async ({ page }) => {
+  const editor = await ProcessEditor.openProcess(page);
+  await expect(editor.diagram).not.toBeFocused();
+  await editor.focusDiagram();
```

Comment:
I think I would add here the check too, otherwise it is a little bit unclear that here is an assertion... 🙈